### PR TITLE
Minor beauty/consistensy fixes

### PR DESCRIPTION
**Pull request by KronosTheLate**

### CODE REVIEW

**Comment by KronosTheLate** [comment]
Currently, the inverval is given by [t_0, t_f] in the start of the changed file. The subscript should probably consistently be letters or numbers. Also, if latex works (which I am not sure), the `in` was just missing a backslash, and that would look better than my proposed change.

File: docs/src/basics/NonlinearProblem.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 ## The Three Types of Nonlinear Problems
 
-NonlinearSolve.jl tackles two related types of nonlinear systems:
+NonlinearSolve.jl tackles three related types of nonlinear systems:
 
-1. Interval rootfinding problems. I.e., find the ``t in [t_0, t_f]`` such that `f(t) = 0`.
+1. Interval rootfinding problems. I.e., find the ``t`` in ``[t_0, t_f]`` such that `f(t) = 0`.
```
Comment (ChrisRackauckas) [inline docs/src/basics/NonlinearProblem.md:7]:
```suggestion
1. Interval rootfinding problems. I.e., find the ``t \in [t_0, t_f]`` such that `f(t) = 0`.
```

File: docs/src/basics/NonlinearProblem.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 ## The Three Types of Nonlinear Problems
 
-NonlinearSolve.jl tackles two related types of nonlinear systems:
+NonlinearSolve.jl tackles three related types of nonlinear systems:
 
-1. Interval rootfinding problems. I.e., find the ``t in [t_0, t_f]`` such that `f(t) = 0`.
+1. Interval rootfinding problems. I.e., find the ``t \in [t_0, t_f]`` such that `f(t) = 0`.
```
Comment (ChrisRackauckas) [inline docs/src/basics/NonlinearProblem.md:7]:
```suggestion
1. Interval rootfinding problems. I.e., find the ``t \in [t_0, t_f]`` such that ``f(t) = 0``.
```

File: docs/src/basics/NonlinearProblem.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
...
 2. Systems of nonlinear equations, i.e. find the `u` such that `f(u) = 0`.
```
Comment (ChrisRackauckas) [inline docs/src/basics/NonlinearProblem.md:8]:
```suggestion
2. Systems of nonlinear equations, i.e. find the ``u`` such that ``f(u) = 0``.
```

File: docs/src/basics/NonlinearProblem.md [hunk 4]
```diff
@@ -2,9 +2,9 @@
...
 3. Steady state problems, i.e. find the `u` such that `u' = f(u,t)` has reached steady state,
    i.e. `0 = f(u, ∞)`.
```
Comment (ChrisRackauckas) [inline docs/src/basics/NonlinearProblem.md:10]:
```suggestion
   i.e. ``0 = f(u, ∞)``.
```

**Comment by KronosTheLate** [comment]
You just get things done, don't you ^_^
Thanks!

**Pull request merged**